Client/Client.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+
+namespace RedGear 
+{
+    namespace RedGearClient 
+    {
+        public class Client 
+        {
+            public static VerifySystem.Verify Launch() 
+            {
+                try 
+                {
+                    NetworkSystem.PackageManager.StartConnexion();
+                    return new VerifySystem.SuccessSystem.OkSuccess("hehe", "Content");
```

Comment:
```suggestion
                    return new VerifySystem.SuccessSystem.OkSuccess("Succes", "Content");
```
A modifié pour mettre un vrai message